### PR TITLE
[chore][cleanup] Add ufmt formatter and format code

### PR DESCRIPTION
**Pull request by ebsmothers**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #1139

Differential Revision: [D32036762](https://our.internmc.facebook.com/intern/diff/D32036762)

### CODE REVIEW

**Comment by ebsmothers** [comment]
@ebsmothers has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32036762).

File: .github/workflows/linter_test.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       fail-fast: false
       max-parallel: 3
       matrix:
-        lint: ['flake8', 'black', 'isort']
+        lint: ['flake8', 'black', 'isort', 'ufmt']
```
Comment (apsdehal) [inline .github/workflows/linter_test.yaml:15]: Do you need black and isort now after ufmt?

**Review by apsdehal — APPROVED** [review]
Fix the minor comment before landing.

File: .github/workflows/linter_test.yaml (hunk 1 above)
Comment (ebsmothers) [reply .github/workflows/linter_test.yaml:15]: Good point, we shouldn't need them anymore. Let me remove them

**Comment by ebsmothers** [comment]
@ebsmothers has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D32036762).

**Pull request closed without merging**